sep53/src/lib.rs

```diff
@@ -0,0 +1,162 @@
+use defuse_crypto::{CryptoHash, Curve, Ed25519, Payload, SignedPayload, serde::AsCurve};
+use impl_tools::autoimpl;
+use near_sdk::{env, near};
+use serde_with::serde_as;
+
+/// See [SEP-53](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0053.md)
+#[near(serializers = [json])]
+#[serde(rename_all = "camelCase")]
+#[derive(Debug, Clone)]
+#[must_use]
+pub struct Sep53Payload {
+    pub message: Vec<u8>,
```

Comment:
Hey, this way in JSON it would be serialized as `[1, 2, 3, ...]`, which is unreadable and not compact representation.
Since we only allow string payloads, accept it as a string:
```suggestion
    pub payload: String,
```